packages/react-components/src/components/TypingIndicator.tsx

```diff
@@ -238,7 +240,12 @@ const getIndicatorComponent = (
   );
 
   return (
-    <Stack horizontal className={mergeStyles(typingIndicatorStringStyle, styles?.typingString)} key="typingStringKey">
+    <Stack
+      horizontal
+      data-ui-id={ids.typingIndicator}
+      className={mergeStyles(typingIndicatorStringStyle, styles?.typingString)}
+      key="typingStringKey"
```

Comment:
Tangential to this PR: The `key` is unnecessary. Explicit keys are only really needed in lists (e.g. `Stack.Item`).

Reply:
Didn't add it. It was already there.